# Base X, Base Y, Base Heading, Grabber X, Grabber Y, Grabber Z, Vector X, Vector Y, Vector Z
waypoint_list: [
#   [0.0,0.0,0.0, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
#   [0.0,1.0,0.0, 1.0, 1.0, 0.5, 0.0, 0.0, 0.0, 1.0],
#   [0.0,1.0,0.0, 1.0, 1.0, 0.7, 0.0, 0.0, 0.0, 1.0]
# ]





# [0,   0, 0],
# [1,  -1, 0.78540],
# [2,-1.1, 1.57080],
# [3,-1.1, 1.57080],
# [4,-1.1, 1.57080],
# [5,-1.1, 1.57080],
# [5.8, 0, 3.14159],
# [5, 1.1,-1.57080],
# [4, 1.1,-1.57080],
# [3, 1.1,-1.57080],
# [2, 1.1,-1.57080],
# [1,   1,-0.78540],
# [1.2, 0, 0]]


# 0.5m Set
 #[1.58152,-0.6908,-1.57,   1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[2.25908,-1.00582,-1.57,  1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[3.4498,-0.947264,-1.57,  1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[4.09529,-0.85792,-1.57,  1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[4.73901,-0.755544,-1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[5.264525,0.0,3.14159,    1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[4.09529,0.85792,1.57,    1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[3.4498,0.947263,1.57,    1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[2.25908,1.00582,1.57,    1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[1.58152,0.6908,1.57,     1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0],
 #[0.971947,0.0,0.0,        1.0, 0.0, 0.5, 0.0, 0.0, 0.0, 1.0]
 #]

# Arm Stance Set
#  BaseX,    BaseY,  Rot, GrX, GrY, GrZ, VeX, VeY, VeZ
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 1.0, 0.0, 0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,-1.0, 0.0, 0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0, 1.0, 0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0,-1.0, 0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0, 0.0, 1.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0, 0.0,-1.0],
# ]

# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0,1.0,0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0,0.707,0.707],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0,0.0,1.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0,-0.707,0.707],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0,-1.0,0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 1.0,0.0,0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.707,0.707,0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.0,1.0,0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, -0.707,0.707,0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, -1.0,0.0,0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,-0.707,-0.707,0.707],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,-0.707,-0.707,0.707],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,-0.707,-0.707,0.707],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,-0.707,-0.707,0.707],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,-0.707,-0.707,0.707],

# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,0.015467,-0.262283,0.964867],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,0.024312,-0.230724,0.972715],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,0.03337,-0.231742,0.972205],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,0.1328,-0.990872,-0.023168],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,0.101064,-0.989746,-0.100944],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5,0.099446,-0.993726,-0.051168],



# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, -0.707,-0.707,0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, -0.707,0.707,0.0],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, 0.577,0.577,0.577],
# [0.00, 0.00, 0.00, 0.6, 0.0, 0.5, -0.577,0.577,0.577],


# Special, the lower half has offsets added, Offset set
# [3.44980,-1.197933,1.57,3.799198,-0.363977,0.640667,-0.002315,-0.231187,0.972907],
# [3.44980,-1.197933,1.57,3.814919,-0.240742,0.673913,0.015467,-0.262283,0.964867],
# [3.44980,-1.197933,1.57,3.822778,-0.179128,0.690536,0.024312,-0.230724,0.972715],
# [3.44980,-1.197933,1.57,3.830235,-0.120076,0.702275,0.03337,-0.231742,0.972205],
# [3.44980,-1.197933,1.57,3.838098,-0.058463,0.718899,0.033568,-0.264316,0.963852],
# [3.44980,-1.197933,1.57,3.784092,-0.481483,0.616392,-0.020092,-0.199793,0.979632],
# [3.44980,-1.197933,1.57,3.842027,-0.027654,0.72721,0.041166,-0.21101,0.976617],
# [3.44980,-1.497933,1.57,3.727808,-0.493438,0.523776,0.063459,-0.996479,0.054801],
# [3.44980,-1.497933,1.57,3.789166,-0.481228,0.616559,0.047738,-0.993919,0.099228],
# [3.44980,-1.497933,1.57,3.79478,-0.490749,0.268035,0.114638,-0.993336,-0.011873],
# [3.44980,-1.497933,1.57,3.904041,-0.480265,0.445738,0.101208,-0.994842,-0.006872],
# [4.09529,-1.108589,1.57,4.008505,-0.470043,0.623331,-0.005373,-0.194792,0.98083],
# [4.09529,-1.108589,1.57,4.038721,-0.235022,0.67188,0.019564,-0.206892,0.978168],
# [4.09529,-1.108589,1.57,4.046178,-0.175974,0.683619,0.033134,-0.198912,0.979457],
# [4.09529,-1.108589,1.57,4.053641,-0.116926,0.695359,0.033368,-0.231744,0.972204],
# [4.09529,-1.108589,1.57,4.061499,-0.055308,0.711982,0.041158,-0.211074,0.976603],
# [4.09529,-1.408589,1.57,4.013298,-0.469775,0.62344,0.083815,-0.996481,0.000133],
# [4.09529,-1.408589,1.57,4.031641,-0.459356,0.282898,0.1328,-0.990872,-0.023168],
# [4.09529,-1.408589,1.57,4.110396,-0.455893,0.413376,0.101064,-0.989746,-0.100944],
# [4.09529,-1.408589,1.57,4.134521,-0.457675,0.455004,0.099446,-0.993726,-0.051168],
# ]

# Non-Offset Set
# [2.84980,-0.997933,0.0,3.549128,-0.436604,0.405216,0.101208,-0.994842,-0.006872],
# [2.84980,-0.997933,0.0,3.4498,-0.446136,0.243668,0.114638,-0.993336,-0.011873],
# [2.84980,-0.997933,0.0,3.665128,-0.417596,0.25718,0.1328,-0.990872,-0.023168],
# [2.84980,-0.997933,0.0,3.758656,-0.416068,0.41364,0.099446,-0.993726,-0.051168],
# [2.84980,-0.997933,0.0,3.736724,-0.414448,0.375796,0.101064,-0.989746,-0.100944],
# [2.84980,-0.997933,0.0,3.648452,-0.427068,0.566764,0.083815,-0.996481,0.000133],
# [2.84980,-0.997933,0.0,3.388916,-0.44858,0.47616,0.063459,-0.996479,0.054801],
# [2.84980,-0.997933,0.0,3.444696,-0.43748,0.560508,0.047738,-0.993919,0.099228],
# [3.44980,-0.997933,0.0,3.48918,-0.053148,0.653544,0.033568,-0.264316,0.963852],
# [3.44980,-0.997933,0.0,3.468108,-0.218856,0.612648,0.015467,-0.262283,0.964867],
# [3.44980,-0.997933,0.0,3.685128,-0.106296,0.632144,0.033368,-0.231744,0.972204],
# [3.44980,-0.997933,0.0,3.482032,-0.10916,0.638432,0.03337,-0.231742,0.972205],
# [3.44980,-0.997933,0.0,3.475252,-0.162844,0.62776,0.024312,-0.230724,0.972715],
# [3.44980,-0.997933,0.0,3.453816,-0.330888,0.582424,-0.002315,-0.231187,0.972907],
# [3.44980,-0.997933,0.0,3.692272,-0.05028,0.647256,0.041158,-0.211074,0.976603],
# [3.44980,-0.997933,0.0,3.492752,-0.02514,0.6611,0.041166,-0.21101,0.976617],
# [3.44980,-0.997933,0.0,3.671564,-0.213656,0.6108,0.019564,-0.206892,0.978168],
# [3.44980,-0.997933,0.0,3.678344,-0.159976,0.621472,0.033134,-0.198912,0.979457],
# [3.44980,-0.997933,0.0,3.440084,-0.437712,0.560356,-0.020092,-0.199793,0.979632],
# [3.44980,-0.997933,0.0,3.644096,-0.427312,0.566664,-0.005373,-0.194792,0.98083],
# [3.49529,-0.908589,0.0,3.830252,-0.412924,0.532256,0.062436,-0.994929,-0.078859],
# [3.49529,-0.908589,0.0,3.852184,-0.414548,0.5701,0.062436,-0.994927,-0.078874],
# [3.49529,-0.908589,0.0,4.05592,-0.402024,0.573436,0.069898,-0.994084,-0.083129],
# [3.49529,-0.908589,0.0,4.012052,-0.39878,0.497752,0.075654,-0.993119,-0.089388],
# [3.49529,-0.908589,0.0,3.968184,-0.395536,0.422064,0.094927,-0.991116,-0.093155],
# [3.49529,-0.908589,0.0,3.880452,-0.389052,0.270692,0.117677,-0.98777,-0.102291],
# [3.49529,-0.908589,0.0,3.92432,-0.392292,0.34638,0.115588,-0.986956,-0.112058],
# [3.49529,-0.908589,0.0,4.218448,-0.379548,0.502636,0.095305,-0.982894,-0.157595],
# [3.49529,-0.908589,0.0,4.196512,-0.377928,0.464788,0.106179,-0.980868,-0.163167],
# [3.49529,-0.908589,0.0,4.177396,-0.371964,0.429728,0.116462,-0.97871,-0.169007],
# [3.49529,-0.908589,0.0,4.155464,-0.370344,0.391884,0.127056,-0.97644,-0.174417],
# [3.49529,-0.908589,0.0,4.136344,-0.364376,0.35682,0.137105,-0.974045,-0.180106],
# [3.49529,-0.908589,0.0,4.114412,-0.362756,0.318972,0.147421,-0.971574,-0.185232],
# [3.49529,-0.908589,0.0,4.095292,-0.356792,0.283908,0.158776,-0.955379,-0.24908],
# [4.09529,-0.908589,0.0,3.88822,-0.103428,0.62586,0.033372,-0.231718,0.97221],
# [4.09529,-0.908589,0.0,3.891796,-0.07542,0.633412,0.041165,-0.211016,0.976615],
# [4.09529,-0.908589,0.0,3.874996,-0.207388,0.607492,0.021532,-0.179473,0.983527],
# [4.09529,-0.908589,0.0,3.88144,-0.157108,0.615184,0.026976,-0.176437,0.983942],
# [4.09529,-0.908589,0.0,3.861264,-0.314216,0.58542,0.007591,-0.177803,0.984037],
# [4.09529,-0.908589,0.0,3.895016,-0.05028,0.63726,0.048626,-0.157204,0.986368],
# [4.09529,-0.908589,0.0,4.091316,-0.10056,0.619572,0.035861,-0.156225,0.98707],
# [4.09529,-0.908589,0.0,4.078432,-0.20112,0.60418,0.018596,-0.157332,0.987371],
# [4.09529,-0.908589,0.0,3.848376,-0.414776,0.570028,-0.007059,-0.150413,0.988598],
# [4.09529,-0.908589,0.0,4.05266,-0.402244,0.573396,0.004752,-0.144877,0.989438],
# ]

# Reduced Set
[2.84980,-0.997933,0.0,3.388916,-0.44858,0.47616,0.063459,-0.996479,0.054801],
[2.84980,-0.997933,0.0,3.444696,-0.43748,0.560508,0.047738,-0.993919,0.099228],

[3.44980,-0.997933,0.0,3.48918,-0.053148,0.653544,0.033568,-0.264316,0.963852],
[3.44980,-0.997933,0.0,3.468108,-0.218856,0.612648,0.015467,-0.262283,0.964867],

[3.49529,-0.908589,0.0,3.830252,-0.412924,0.532256,0.062436,-0.994929,-0.078859],
[3.49529,-0.908589,0.0,3.852184,-0.414548,0.5701,0.062436,-0.994927,-0.078874],

[4.09529,-0.908589,0.0,3.88822,-0.103428,0.62586,0.033372,-0.231718,0.97221],
[4.09529,-0.908589,0.0,3.891796,-0.07542,0.633412,0.041165,-0.211016,0.976615],
]

# [1.49214,-0.805030, 0.90, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [1.94565,-1.173537, 1.20, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [2.25908,-1.256489, 1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [2.80120,-1.257233, 1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [3.44980,-1.197933, 1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [4.09529,-1.108589, 1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [4.73901,-1.006213, 1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [5.51928,-0.000388, 3.14, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [4.73901, 1.006213,-1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [4.09529, 1.108589,-1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [3.44980, 1.197932,-1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [2.80120, 1.257233,-1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [2.25908, 1.256489,-1.57, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [1.94565, 1.173537,-1.20, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [1.49214, 0.805029,-0.90, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0],
# [0.71748, 0.000000, 0.00, 1.0, 0.0, 0.5, 0.0, 0.0, 0.0]
# ]





  # [0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 ],
  # [0.1, 1.1, 2.1, 3.1, 4.1, 5.1, 6.1, 7.1, 8.1 ],
  # [0.2, 1.2, 2.2, 3.2, 4.2, 5.2, 6.2, 7.2, 8.2 ],
  # [0.3, 1.3, 2.3, 3.3, 4.3, 5.3, 6.3, 7.3, 8.3 ],
  # [0.4, 1.4, 2.4, 3.4, 4.4, 5.4, 6.4, 7.4, 8.4 ],
  # [0.5, 1.5, 2.5, 3.5, 4.5, 5.5, 6.5, 7.5, 8.5 ]
  # {base_pos_x: 1.0, base_pos_y: 1.0, eff_pos_x: 1.0, eff_pos_y: 1.0, eff_pos_z: 0.0, eff_orint_x: 0, eff_orint_y: 0, eff_orint_z: 0.0, eff_orint_w: 1.0},
  # {base_pos_x: 1.0, base_pos_y: 1.0, eff_pos_x: 2.0, eff_pos_y: 2.0, eff_pos_z: 0.0, eff_orint_x: 0, eff_orint_y: 0, eff_orint_z: 0.0, eff_orint_w: 1.0},
  # {base_pos_x: 1.0, base_pos_y: 1.0, eff_pos_x: 3.0, eff_pos_y: 3.0, eff_pos_z: 0.0, eff_orint_x: 0, eff_orint_y: 0, eff_orint_z: 0.0, eff_orint_w: 1.0},
  # {base_pos_x: 1.0, base_pos_y: 1.0, eff_pos_x: 4.0, eff_pos_y: 4.0, eff_pos_z: 0.0, eff_orint_x: 0, eff_orint_y: 0, eff_orint_z: 0.0, eff_orint_w: 1.0}
# ]